docs/config/dfdaemon_properties.md

```diff
@@ -0,0 +1,77 @@
+# Customize dfdaemon properties
+
+This topic explains how to customize the dragonfly dfdaemon startup parameters.
+
+**NOTE**: Properties holds all configurable properties of dfdaemon including `dfget` properties. By default, dragonfly config files locate `/etc/dragonfly`. You can create `dfdaemon.yml` for configuring dfdaemon startup params. When deploying with Docker, you can mount the default path when starting up dfdaemon image with `-v`.
```

Comment:
1. locate -> locate at
2. starting up dfdaemon image -> running dfclient image

I think we don't have an image called dfdaemon.